server/src/test/java/com/objectcomputing/checkins/services/file/FileControllerTest.java

```diff
@@ -55,237 +60,117 @@ void deleteTestFile() {
+        UUID testCheckinId = UUID.randomUUID();
+        when(fileServices.uploadFile(any(UUID.class), any(CompletedFileUpload.class))).thenReturn(expectedResponse);
 
-        assertEquals(HttpStatus.BAD_REQUEST,  responseException.getStatus());
-        assertEquals("You are not authorized to perform this operation", error);
-    }
+        final HttpRequest<?> request = HttpRequest.POST(String.format("/%s", testCheckinId), MultipartBody.builder()
+                                        .addPart("file", testFile).build())
+                                        .basicAuth("some.email", MEMBER_ROLE)
+                                        .contentType(MULTIPART_FORM_DATA);
+        final HttpResponse<FileInfoDTO> response = client.toBlocking().exchange(request, FileInfoDTO.class);
 
-    @Test
-    public void testFindByCheckInId() {
-
-        //arrange
-        MemberProfile memberProfileOfPDL = createADefaultMemberProfile();
-        MemberProfile memberProfileOfUser = createADefaultMemberProfileForPdl(memberProfileOfPDL);
-        CheckIn checkin  = createADefaultCheckIn(memberProfileOfUser, memberProfileOfPDL);
-
-        final HttpRequest<?> requestForPostEndpoint = HttpRequest.POST(String.format("/%s", checkin.getId()), MultipartBody.builder()
-                                                        .addPart("file", testFile).build())
-                                                        .basicAuth(memberProfileOfUser.getWorkEmail(), MEMBER_ROLE)
-                                                        .contentType(MULTIPART_FORM_DATA);
-        final HttpResponse<FileInfoDTO> responseFromPostEndpoint = client.toBlocking().exchange(requestForPostEndpoint, FileInfoDTO.class);
-        assertEquals(HttpStatus.OK, responseFromPostEndpoint.getStatus());
-        FileInfoDTO resultFromPost = responseFromPostEndpoint.getBody().get();
-
-        //act
-        final HttpRequest<?> requestForFindById = HttpRequest.GET(String.format("?id=%s", checkin.getId()))
-                                                    .basicAuth(memberProfileOfPDL.getWorkEmail(), MEMBER_ROLE);
-        final HttpResponse<Set<FileInfoDTO>> responseFromFindById = client.toBlocking().exchange(requestForFindById, Argument.setOf(FileInfoDTO.class));
-        FileInfoDTO result = Objects.requireNonNull(responseFromFindById.body()).iterator().next();
-
-        //assert
-        assertNotNull(result);
-        assertEquals(HttpStatus.OK, responseFromFindById.getStatus());
-        assertEquals(resultFromPost.getFileId(), result.getFileId());
-        assertEquals(resultFromPost.getCheckInId(), result.getCheckInId());
-        assertEquals(resultFromPost.getName(), result.getName());
+        assertNotNull(response);
+        verify(fileServices, times(1)).uploadFile(any(UUID.class), any(CompletedFileUpload.class));
     }
 
     @Test
-    public void testFindByCheckInIdFailsForUnrelatedUser() {
+    public void testUploadEndpointFailsForInvalidFile() {
```

Comment:
This test failed for me.  `(No such file or directory)` vs `(The system cannot find the path specified)`

Reply:
Not sure why you are seeing this. Could you try once after the latest commit? I am not seeing this on my local and looks like tests pass on Github actions as well.